src/display/annotation_layer.js

```diff
@@ -1217,6 +1338,13 @@ class ChoiceWidgetAnnotationElement extends WidgetAnnotationElement {
       }
     }
 
+    selectElement.addEventListener("resetform", event => {
+      const defaultValue = this.data.defaultFieldValue;
+      Array.prototype.forEach.call(selectElement.options, option => {
```

Comment:
Unless I'm mistaken, I believe that you should be able to iterate through the options with a `for...of` loop instead. Assuming that works, let's do that instead since it feels nicer than having to use Array.forEach in this way.